apps/web/src/stories/Button.stories.ts

```diff
@@ -11,33 +11,46 @@ const meta = {
   argTypes: {
     size: {
       control: {
-        type: 'radio',
-        options: ['S', 'M', 'L'], // 사용 가능한 타입들을 선택 옵션으로 제공합니다.
+        type: 'text',
+        // type: 'radio',
+        // options: ['S', 'M', 'L'], // 사용 가능한 타입들을 선택 옵션으로 제공합니다.
       },
     },
     variant: {
       control: {
```

Comment:
```suggestion
    variant: {
      options: ['primary', 'outline'], // 사용 가능한 타입들을 선택 옵션으로 제공합니다.
      control: {
          type: 'select'
```
option 한단계 상위 레벨에 주니까 되네요 [참고](https://storybook.js.org/docs/react/essentials/controls)

Reply:
뚀잉!